### PR TITLE
Run clang format against files under tools/ and db_stress_tool/

### PR DESCRIPTION
**Pull request by siying**
Summary: Some lines of .h and .cc files are not properly fomatted. Clear them up with clang format.

Test Plan: Watch existing CI to pass

### CODE REVIEW

**Comment by siying** [comment]
I manually changed line wrapping for many strings after clang formatting.

File: tools/simulated_hybrid_file_system.cc [hunk 1]
```diff
@@ -6,13 +6,12 @@
...
 #ifndef ROCKSDB_LITE
 
-#include "tools/simulated_hybrid_file_system.h"
-
 #include <algorithm>
 #include <sstream>
 #include <string>
 
 #include "rocksdb/rate_limiter.h"
+#include "tools/simulated_hybrid_file_system.h"
```
Comment (ajkr) [inline tools/simulated_hybrid_file_system.cc:14]: This one looks odd - usually the header file corresponding to the source file gets included first

**Review by ajkr — APPROVED** [review]
LGTM

File: tools/simulated_hybrid_file_system.cc (hunk 1 above)
Comment (siying) [reply tools/simulated_hybrid_file_system.cc:14]: Yes. claim to clang-format?

**Pull request closed without merging**